pkg/pillar/cmd/zedrouter/zedrouter.go

```diff
@@ -2137,3 +2182,46 @@ func addAppInstanceOverlayRoute(
 			status.DisplayName)
 	}
 }
+
+// getSSHPublicKeys : returns decrypted SSH public keys
+func getSSHPublicKeys(ctx *zedrouterContext, dc *types.AppNetworkConfig) []string {
+	// TBD: add ssh keys into cypher block
+	return nil
+}
+
+// getCloudInitUserData : returns decrypted cloud-init user data
+func getCloudInitUserData(ctx *zedrouterContext,
```

Comment:
This function needs to change once https://github.com/lf-edge/eve/pull/2000 goes it.

Reply:
If I correctly understood, inside https://github.com/lf-edge/eve/pull/2007 you made internal changes.

Reply:
Correct. Just need the tag on the type.